### PR TITLE
Add permissions-policy header

### PR DESCRIPTION
**Pull request by IniZio**
ref #4018

### CODE REVIEW

File: pkg/util/httputil/static_security_headers.go [hunk 1]
```diff
@@ -7,6 +7,14 @@ import (
 func StaticSecurityHeaders(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("X-Content-Type-Options", "nosniff")
+		/**
```
Comment (louischan-oursky) [inline pkg/util/httputil/static_security_headers.go:10]:
```
package httputil

type PermissionsPolicyDirective string

const (
	PermissionsPolicyDirectiveAutoplay = "autoplay"
)

type PermissionsPolicyAllowlist string

const (
	PermissionsPolicyAllowlistAll PermissionsPolicyAllowlist = "*"
	PermissionsPolicyAllowlistNone PermissionsPolicyAllowlist = ""
	PermissionsPolicyAllowlistSelf PermissionsPolicyAllowlist = "self"
)

type PermissionsPolicyPolicy struct {
	Directive PermissionsPolicyDirective
	Allowlist PermissionsPolicyAllowlist
}

var DefaultPermissionsPolicy []PermissionsPolicyPolicy = []PermissionsPolicyPolicy{
	// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives
}

type HTTPPermissionsPolicy []PermissionsPolicyPolicy
```

File: pkg/util/httputil/permissions_policy.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives
+type PermissionsPolicyDirective string
+
+const (
+	PermissionsPolicyDirectiveAccelerometer               PermissionsPolicyDirective = "accelerometer"
+	PermissionsPolicyDirectiveAmbientLightSensor          PermissionsPolicyDirective = "ambient-light-sensor"
+	PermissionsPolicyDirectiveAutoplay                    PermissionsPolicyDirective = "autoplay"
+	PermissionsPolicyDirectiveBattery                     PermissionsPolicyDirective = "battery"
+	PermissionsPolicyDirectiveBluetooth                   PermissionsPolicyDirective = "bluetooth"
```
Comment (louischan-oursky) [inline pkg/util/httputil/permissions_policy.go:16]: Missing `browsing-topics` after `bluetooth`

File: pkg/util/httputil/permissions_policy.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	PermissionsPolicyDirectiveCamera                      PermissionsPolicyDirective = "camera"
+	PermissionsPolicyDirectiveDisplayCapture              PermissionsPolicyDirective = "display-capture"
+	PermissionsPolicyDirectiveDocumentDomain              PermissionsPolicyDirective = "document-domain"
+	PermissionsPolicyDirectiveEncryptedMedia              PermissionsPolicyDirective = "encrypted-media"
+	PermissionsPolicyDirectiveExecutionWhileNotRendered   PermissionsPolicyDirective = "execution-while-not-rendered"
+	PermissionsPolicyDirectiveExecutionWhileOutOfViewport PermissionsPolicyDirective = "execution-while-out-of-viewport"
+	PermissionsPolicyDirectiveFullscreen                  PermissionsPolicyDirective = "fullscreen"
```
Comment (louischan-oursky) [inline pkg/util/httputil/permissions_policy.go:23]: Missing `gamepad` after `fullscreen`

Comment (IniZio) [reply pkg/util/httputil/permissions_policy.go:24]: Updated, should have 36 items now matching doc's list 🙏

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**